src/helpers/validation.ts

```diff
@@ -245,13 +245,6 @@ export const validateServiceJourney = (
   const validDayTimes = (sj.dayTypes?.[0]?.daysOfWeek?.length ?? 0) > 0;
   const { isValid } = validateTimes(sj.passingTimes ?? [], intl);
   const validDayTypes = validateDayTypes(sj.dayTypes);
-  console.log('validateServiceJourney', {
-    sj,
-    validName: !isBlankName,
-    validDayTimes,
-    validPassingTimes: isValid,
-    validDayTypes,
-  });
```

Comment:
Was cleaning up own console log and thought to take this one along, no idea if it actually was serving any special purpose